src/reuse/_comment.py

```diff
@@ -450,13 +450,17 @@ class UncommentableCommentStyle(EmptyCommentStyle):
     ".applescript": AppleScriptCommentStyle,
     ".bash": PythonCommentStyle,
     ".bat": BatchFileCommentStyle,
+    ".bb": PythonCommentStyle,
+    ".bbappend": PythonCommentStyle,
+    ".bbclass": PythonCommentStyle,
     ".bib": BibTexCommentStyle,
     ".c": CCommentStyle,
     ".cl": LispCommentStyle,
     ".clj": LispCommentStyle,
     ".cljc": LispCommentStyle,
     ".cljs": LispCommentStyle,
     ".cmake": PythonCommentStyle,  # TODO: Bracket comments not supported.
+    ".conf": PythonCommentStyle,
```

Comment:
I am not sure whether we can assume that all `.conf` files can be commented in the Python style. Is there some standard for this?

Reply:
I searched more and couldn't find an actual standard, only that it's widely used with this style.

Any ideas where I could search for such a standard?

Reply:
Yeah, and even if there was one, it seems there are also cases where this would break. My suggestion would be to not define this, same for other files like `cfg` or so.

Reply:
Understood, I'll resend without the `.conf` change.

Reply:
Excellent, thanks!